includes/amp-post-template-functions.php

```diff
@@ -54,6 +55,20 @@ function amp_post_template_add_fonts( $amp_template ) {
 	}
 }
 
+/**
+ * Add block styles for core blocks and third-party blocks.
+ *
+ * @param AMP_Post_Template $amp_template Template.
+ */
+function amp_post_template_add_block_styles( $amp_template ) {
```

Comment:
The `$amp_template` arg can be removed, no?

Reply:
It can be removed, yes, although it is indeed being passed-in.

I'll remove it.

Reply:
Resolved via https://github.com/ampproject/amp-wp/pull/4299/commits/0baf7b90c13de89b585fd91fd274b260def57b6e